_posts/2023-07-04-LLM-Transformers.markdown

```diff
@@ -0,0 +1,264 @@
+---
+layout: postv2
+font: serif
+title: "LLM Transformers"
+description: "Empower LLM models with Transformers and Agents"
```

Comment:
LLM Models is Large Language Models Models

Reply:
Sorry Hamo, what needs to be fixed here?